query-service/src/integrationTest/resources/view-generator/application.conf

```diff
@@ -0,0 +1,23 @@
+service.name = view-generators
+service.admin.port = 8099
+
+main.class = org.hypertrace.core.viewgenerator.service.MultiViewGeneratorLauncher
```

Comment:
Do we have to copy this? Can't this be also used from the image?

Reply:
This pr would allow us to use the same config file https://github.com/hypertrace/hypertrace-ingester/pull/146